src/config.json

```diff
@@ -37,50 +37,59 @@
     },
     "dependencies": {
         "anymatch": "1.3.0",
-        "aws-sdk": "^2.19.0",
+        "autoprefixer-core": "5.1.8",
+        "aws-sdk": "^2.2.9",
         "chokidar": "1.6.0",
+        "lodash": "4.15.0",
         "glob": "7.0.6",
         "grunt": "0.4.5",
-        "grunt-cleanempty": "1.0.3",
         "grunt-cli": "0.1.9",
+        "jasmine-node": "1.11.0",
+        "jshint": "2.1.4",
+        "grunt-jasmine-node": "0.1.0",
+        "phantomjs": "1.9.18",
+        "grunt-lib-phantomjs": "0.3.0",
+        "grunt-eslint": "18.1.0",
+        "grunt-contrib-watch": "0.4.3",
+        "grunt-contrib-jasmine": "0.4.2",
+        "grunt-template-jasmine-requirejs": "0.1.0",
+        "grunt-contrib-cssmin": "0.6.0",
         "grunt-contrib-clean": "0.4.1",
-        "grunt-contrib-compress": "^1.4.1",
         "grunt-contrib-concat": "0.3.0",
+        "grunt-contrib-compress": "^1.4.1",
         "grunt-contrib-copy": "0.4.1",
-        "grunt-contrib-cssmin": "0.6.0",
         "grunt-contrib-htmlmin": "0.1.3",
-        "grunt-contrib-jasmine": "0.4.2",
+        "grunt-contrib-jshint": "0.6.0",
+        "grunt-exec": "0.4.6",
+        "grunt-postcss": "0.3.0",
         "grunt-contrib-less": "1.4.0",
         "grunt-contrib-requirejs": "0.4.1",
         "grunt-contrib-uglify": "2.1.0",
-        "grunt-contrib-watch": "0.4.3",
-        "grunt-eslint": "18.1.0",
-        "grunt-exec": "^1.0.1",
-        "grunt-jasmine-node": "0.1.0",
-        "grunt-lib-phantomjs": "0.3.0",
+        "grunt-git": "^0.3.4",
+        "grunt-npm": "git://github.com/sedge/grunt-npm.git#branchcheck",
```

Comment:
NOTE: we can do that, but not in this bug.  This bug isn't actually changing any npm deps, it just got a cached version of what's in `package.json` due to a build config (which no longer happens on `master` btw).